nevergrad/functions/gym/multigym.py

```diff
@@ -557,6 +561,18 @@ def __init__(
 
         # Create the parametrization.
         parametrization = parameter.Array(shape=shape).set_name("ng_default")
+        if sparse_limit is not None:
+            parametrization1 = parameter.Array(shape=shape)
+            parametrization2 = parameter.Array(shape=shape)  # 0/1 weights for enabling/disabling
+            parametrization2.set_integer_casting()
+            parametrization2.set_bounds(0, 1)
+            parametrization = ng.p.Dict(  # type: ignore
+                weights=parametrization1,
+                enablers=parametrization2,
+            )
+            parametrization.set_name("ng_sparse" + str(sparse_limit))
```

Comment:
not sure if the name is useful in the end, since sparse_limit will already appears in the report anyway. shouldnt be harmful either, so up to you

Reply:
Ok it was working with the good old Arrays, but I fail to make it work with Instrumentation and Choice or Tuple and Choice.